find-ttyusb.c:find_ttyusb:55:FOUND: </dev/ttyUSB0>
opened tty port </dev/ttyUSB0>.
pi-install: tty-usb=</dev/ttyUSB0> program=<./tests-2.0/part1-test3-bug.bin> about to boot
HASH: simple_boot: sending 12904 bytes, crc32=d61d35ce
waiting for a start
putting code
bootloader: Done.
listening on ttyusb=</dev/ttyUSB0>
test3: use after free
TRACE:going to check heap
TRACE:ERROR:block 1 [0x17fef8] corrupted at offset 0
TRACE:	logical block id=1, [addr=0x17fef8] nbytes=4
TRACE:	Block allocated at: tests-2.0/part1-test3-bug.c:notmain:9
TRACE:	Wrote block after free!
TRACE:checked 1 blocks, detected 1 errors
TRACE:SUCCESS: detected corruption in 1 [0x17fef8]
DONE!!!

Saw done

bootloader: pi exited.  cleaning up
